### ISSUE TITLE
Conflicts between different versions of a package are shown

### ISSUE THREAD

**Issue by AltGr** [OCamlPro/ows#15]
At the end of http://ows.irill.org/latest/today/summary.html#conflicts , conflicts between different versions of cstructs are shown, which is weird. Possibly the same issue as https://github.com/ocaml/opam/issues/1703, which would take its root in dose ?

**Comment by abate** [comment]
yes, the conflicts regarding cstructs are mostly related to self conflicts. If I understand correctly, the idea in #1703  is that since self-conflicts are "implicit", they should not be shown to the user. But if a package fails to install because of a self conflict, what kind of explanation should we give to the final user ?

**Comment by AltGr** [comment]
After some more thought, I get it: doesn't sound easy to explain better indeed... here the real issue must be something like:
- a -> b -> c > v
- a -> d -> c < v

the full explanation would be to lift the conflict, but we would have "b conflicts with d" _implicitely_ ;
or maybe more clearly, `b conflicts with c <= v` (or equivalently `d conflicts with c >= v`) ...

Not sure introducing implicit conflicts is a good idea, but I can't find a better explanation here.

**Comment by AltGr** [comment]
I thought some more about these conflicts:
- "`b` depends on `(c > v)`" actually implies "`b` **conflicts with** `(c <= v)`"

It would actually seem fair and consistent that this is treated in the same way, as a conflict, in dose and the reports. Would that make these results clearer, though ?
